pkg/translate/transform.go

```diff
@@ -53,6 +53,22 @@ func langTagFromProto(s string) (language.Tag, error) {
 	return l, nil
 }
 
+// langTagsFromProto converts a slice of strings to []language.Tag.
+func langTagsFromProto(langTags []string) ([]language.Tag, error) {
```

Comment:
i guess you could change generic function, so if input is pointer then output is also list with pointers, and vice versa
```go
// sliceFromProto converts a slice of type T to a slice of type R using the provided elementFromProto function.
func sliceFromProto[T any, R any](slice []T, elementFromProto func(T) (R, error)) ([]R, error) {
...
}

// langTagsFromProto converts a slice of strings to []language.Tag.
func langTagsFromProto(langTags []string) ([]language.Tag, error) {
	return sliceFromProto(langTags, langTagFromProto)
}
```

Reply:
I think I might have to create another generic function, function signatures are a bit different for service transformations.